content/fair/interoperable/index.md

```diff
@@ -0,0 +1,141 @@
+
+GitHub as the [repository platform for Galaxy’s code](https://github.com/galaxyproject/galaxy) is a central exchange point for developers at any level of experience, by relying on the built-in tools for interaction and contributions (issues, pull requests, ...). The reaction time is commonly fast, mostly below 24h and in practice help is voluntarily provided also outside business hours and on weekends or holidays; this is partly due to the international scope of the community across time zones. 
+
+Recently, a lightweight questionnaire for user feedback has been integrated into Galaxy Europe’s web interface, being presented occasionally when using tools.
+
+In terms of tutorials and training, videos (screen casts) are available on Youtube, but majorly educative resources are generated and presented with the [Galaxy Training Network (GTN)](https://training.galaxyproject.org), which is web-based and uses Galaxy itself as technical backend and interactive user interface. Galaxy and GTN support Bioschemas annotation to enhance the findability of different assets, with GTN even providing a [dedicated tutorial](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/fair-gtn/tutorial.html) on how to generate FAIR training materials.
+
+## Legal framework, funding, and governance 
+
+### Legal framework
+
+Galaxy is available under a small range of licenses:
+* Web contents of usegalaxy.eu are published as Creative Commons Zero v1.0 (CC0-v1.)
+* Work on the [codebase](https://github.com/galaxyproject/galaxy) contributed from 2021-04-07 onwards is licensed under the MIT License
+* Work on the [codebase](https://github.com/galaxyproject/galaxy) contributed before 2021-04-07 is licensed under the Academic Free License, v3.0
+* Further details of underlying licenses: https://github.com/galaxyproject/galaxy/blob/dev/LICENSE.txt 
+* Every single tool (currently ~3000) has its own license, which is annotated as part of its conda package or container.
+
+### Privacy/Ethics policy
+
+The Galaxy framework has a GDPR configuration option, which will make sure that a deployed instance is GDPR-compliant (to the best of our knowledge). This option is enabled at the European Galaxy server (and related resources). Please read more about it at:
+https://usegalaxy-eu.github.io/gdpr/
+and
+https://docs.galaxyproject.org/en/master/admin/special_topics/gdpr_compliance.html 
+
+### Funding & sustainability plan
+
+The European Galaxy project is embedded in national and international funding streams. Notebly, this is the German ELIXIR Node and EU funding like EOSC-related projects. More information, specifically for the German partners, can be accessed at https://usegalaxy-eu.github.io/freiburg/projects. However, the European Galaxy project is built by many contributors across Europe and the world and the underlying funding is much more diverse. 
```

Comment:
I guess this first sentence can be removed, more emphasis on the second one ... maybe with a link to more funders ... not sure the hub has such a page yet